maestro-studio/server/src/main/java/maestro/studio/MaestroStudio.kt

```diff
@@ -0,0 +1,35 @@
+package maestro.studio
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import maestro.Maestro
+import org.http4k.core.Method.GET
```

Comment:
I suggest we use Ktor instead. It is more mature and team is more familiar with it

Reply:
updated